shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/config/exception/ShardingSphereConfigurationException.java

```diff
@@ -42,4 +42,14 @@ public ShardingSphereConfigurationException(final String errorMessage, final Obj
     public ShardingSphereConfigurationException(final Exception cause) {
         super(cause);
     }
+
+    /**
+     * Constructs an exception with cause exception and error message. 
+     * 
+     * @param cause cause exception
+     * @param errorMessage error message
+     */
+    public ShardingSphereConfigurationException(final Exception cause, final String errorMessage) {
```

Comment:
yes, can replace. in the first place, I thought origin exception stack can see datasource which setter method error, but after try, it can't see which method error